packages/react/src/components/table/table.tsx

```diff
@@ -214,7 +240,7 @@ export interface TableProps<T extends object> {
      * @default medium
      */
     rowSize?: RowSize;
-    selectableRows?: boolean;
+    selectionMode?: SelectionMode;
```

Comment:
C'est un BREAKING CHANGE, il faut ajouter un `!` et documenter le breaking change.

https://www.conventionalcommits.org/en/v1.0.0/#commit-message-with-both--and-breaking-change-footer